data/frame.go

```diff
@@ -41,6 +42,40 @@ type Frame struct {
 	Meta *FrameMeta
 }
 
+type frameJSONFormat struct {
+	Frame []byte `json:"frame"`
```

Comment:
I don't think there should be a struct for this. Rather just to []byte. 

Then also the methods can go on the Frames type, which will end up marshaling to and from [][]byte.

The Frames can wrap the current Marshal method on that type.

Reply:
Can you provide any more info?

I'll try to get that to work.

Reply:
ahh. I see what you mean. I pushed an update. Thanks for the suggestion! For some reason I didn't think to unmarshal directly to a []byte.

Reply:
`Frames` is a type in the `data` package, and is a `[]*Frame` `type Frames []*Frame` (https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go@v0.79.0/data#Frames). (And has similar methods for the arrow marshaling).

`backend.DataResponse` uses that type (https://pkg.go.dev/github.com/grafana/grafana-plugin-sdk-go@v0.79.0/backend#DataResponse).

Often a query response will be Frames, not just a Frame. So my putting the Unmarshal method on that plugin authors will not have to implement the loop.

By not using the struct, I mean not having object with a property. Rather just an array of the base64 encoded binary arrow frames (json.Marshal by default encodes []byte using base64). So a Frame is a string `"QVJST1cxAA"` and Frames are an array of those strings.

Reply:
gotya. That makes sense. Does the change that I added address that? Should I instead unmarshal to / from [][]byte?

It looks like the test I added passes with what's in there now.

Reply:
That changes makes sense. It should be `[]byte`, as that is what will marshal to the base64 encoded string.

The reason I bring up `[][]byte`, as I think we should add the Marshaler/unmarshaler interface to to the Frames type as well with this (unless it isn't need because the having the method on the type takes care of the slice of the type? Not sure....).

Reply:
An nevermind, seems not needed: https://play.golang.org/p/kboXIuiw2ac